microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -0,0 +1,1231 @@
+| [Window management](#window-management) | Window-specific attributes, such as positioning, focus, and keyboard accelerators. |
+| [Composition-based rendering](#composition-based-rendering) | For composition-based WebView2 rendering, use `CoreWebView2Environment` to create a `CoreWebView2CompositionController`.  The `CoreWebView2CompositionController` also implements all the APIs as `CoreWebView2Controller`, but also includes APIs for composition-based rendering. |
+| [User data](#user-data) | Manage the user data folder (UDF), which is a folder on the user's machine.  The UDF contains data related to the host app and WebView2.  WebView2 apps use user data folders to store browser data, such as cookies, permissions, and cached resources. |
+| [Performance and debugging](#performance-and-debugging) | Analyze and debug performance and handle performance-related events for WebView2 controls.  Make your app respond to user input actions quickly and prevent sluggish response.  Manage memory usage and responsiveness. |
+| [Chrome Developer Protocol (CDP)](#chrome-developer-protocol-cdp) | Instrument, inspect, debug, and profile Chromium-based browsers.  The Chrome DevTools Protocol is the foundation for the Microsoft Edge DevTools.  Use the Chrome DevTools Protocol for features that aren't implemented in the WebView2 platform. |
+
+
+<!-- ====================================================================== -->
+## Main classes: Environment, Controller, and Core
+
+<!-- keep sync'd:
+* [Main classes: Environment, Controller, and Core](overview-features-apis.md#main-classes-environment-controller-and-core) in _Overview of WebView2 features and APIs_.
+* [Main classes for WebView2: Environment, Controller, and Core](environment-controller-core.md) - topmost content.
+-->
+
+The `CoreWebView2Environment`, `CoreWebView2Controller`, and `CoreWebView2` classes (or equivalent interfaces) work together so your app can host a WebView2 browser control and access its browser features.  These three large classes expose a wide range of APIs that your host app can access to provide many categories of browser-related features for your users.
+
+*  The `CoreWebView2Environment` class represents a group of WebView2 controls that share the same WebView2 browser process, user data folder, and renderer.  From this `CoreWebView2Environment` class, you create pairs of `CoreWebView2Controller` and `CoreWebView2` instances.
+*  The `CoreWebView2Controller` class is responsible for hosting-related functionality such as window focus, visibility, size, and input, where your app hosts the WebView2 control.
+*  The `CoreWebView2` class is for the web-specific parts of the WebView2 control, including networking, navigation, script, and parsing and rendering HTML.
+
+<!-- / keep sync'd -->
+
+See also:
+* [Main classes for WebView2: Environment, Controller, and Core](environment-controller-core.md)
+
+##### [C#](#tab/c-sharp)
```

Comment:
We have C#, and C++ tabs in this table, but we usually split our apis up based on the api set/runtime? Like win32/.net/winrt. Like maybe there is a place where c++ development can happen in C#, or C++ can be used alongside winrt apis, in which case developers may get confused? I actually don't know enough to advise here, maybe @david-risney has thoughts?

Reply:
Please see how many sets of API Refs we want to link to in the present article's tab-sets:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference#languages-and-frameworks

Reply:
Resolved: we're trying moving toward this model
https://docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#promotions-1
The initial tent. commit, in middle tab, might have a single WinRT placeholder link to top of its API Ref, pending member links.